src/main/pages/che-7/extensions/assembly_using-openshift-connector-in-eclipse-che.adoc

```diff
@@ -0,0 +1,38 @@
+// openshift-connector-overview - upstream only
+
+ifdef::context[:parent-context-of-using-openshift-connector-in-eclipse-che: {context}]
+
+:context: using-openshift-connector-in-eclipse-che
+
+[id="using-openshift-connector-in-eclipse-che_{context}"]
+= Using OpenShift Connector in {prod-short}
+
+OpenShift Connector, also referred to as Visual Studio Code OpenShift Connector  for Red Hat OpenShift, is a plug-in for {prod-short} that provides a convenient and intuitive method for interacting with Red Hat OpenShift 3 or 4 clusters.
+
+OpenShift Connector makes it possible to create, build, and debug applications in the {prod-short} IDE and then deploy the applications directly to the running OpenShift cluster.
+
+OpenShift Connector acts as a GUI for the OpenShift Do (ODO) utility, which aggregates OpenShift CLI (OC) commands into compact units. As such, OpenShift Connector helps new developers who do not have OpenShift background with creating applications and running them on the cloud. Instead of using several `oc` commands, the user creates a new component or service by selecting a preconfigured template, such as a Project, an Application, or a Service; then and deploys it as an OpenShift Component to their cluste
+
+This section provides information about installing, enabling, and basic use of OpenShift Connector plug-in in {prod-short}.
```

Comment:
Hi, I already told you in Slack. I prepare this for downstreaming, where is no Overview section (this will be the main assembly there) this assembly is not included in the upstream, but it carries the xrefs from the Overview assembly.
Hit me on Slack if any more questions will arise :)